build/buildTenDRA.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -ex
+
+VERSION=$1
+if echo ${VERSION} | grep 'trunk'; then
+    VERSION=trunk-$(date +%Y%m%d)
+    BRANCH=main
+else
+    BRANCH=V${VERSION}
+fi
+
+OUTPUT=/root/tendra-${VERSION}.tar.xz
+S3OUTPUT=""
+if echo $2 | grep s3://; then
+    S3OUTPUT=$2
+else
+    OUTPUT=${2-/root/tendra-${VERSION}.tar.xz}
+fi
+
+PREFIX_BOOTSTRAP=$(pwd)/prefix/bootstrap
+PREFIX_REBUILD=$(pwd)/prefix/rebuild
+
+DIR=$(pwd)/tendra
+git clone --depth 1 -b ${BRANCH} https://github.com/tendra/tendra.git ${DIR}
+
+# no -j (currently breaks)
+pmake -C ${DIR} TARGETARCH=x32_64 LIBCVER=GLIBC2_31 OBJ_BPREFIX=${PREFIX_BOOTSTRAP}
```

Comment:
Incidentally setting `LIBCVER` here should no longer be necessary; @FFY00 fixed the automatic identification in https://github.com/tendra/tendra/pull/34